src/egon/data/datasets/gas_neighbours.py

```diff
@@ -0,0 +1,1509 @@
+                "Note",
+                "Category",
+                "Case",
+                "Scenario",
+            ]
+        )
+        .set_index("Node/Line")
+    )
+
+    df_2030 = (
+        df[(df["Parameter"] == "Final demand") & (df["Year"] == 2030)]
+        .rename(columns={"Value": "Value_2030"})
+        .drop(columns=["Parameter", "Year"])
+    )
+
+    df_2040 = (
+        df[(df["Parameter"] == "Final demand") & (df["Year"] == 2040)]
+        .rename(columns={"Value": "Value_2040"})
+        .drop(columns=["Parameter", "Year"])
+    )
+
+    # Conversion GWh/d to MWh/h
+    conversion_factor = 1000 / 24
+
+    df_2035 = pd.concat([df_2040, df_2030], axis=1)
+    df_2035["GlobD_2035"] = (
+        (df_2035["Value_2030"] + df_2035["Value_2040"]) / 2
+    ) * conversion_factor
+    df_2035.loc["NOS0"] = [
+        0,
+        0,
+        Norway_global_demand_1y / 8760,
+    ]  # Manually add Norway demand
+    grouped_demands = df_2035.drop(
+        columns=["Value_2030", "Value_2040"]
+    ).reset_index()
+
+    # choose demands for considered countries
+    return grouped_demands[
+        grouped_demands["Node/Line"].str[:2].isin(countries)
+    ]
+
+
+def import_ch4_demandTS():
+    """Import from the PyPSA-eur-sec run the timeseries of
+    residential rural heat per neighbor country.
+    This timeserie is used to calculate:
+    - the global (yearly) heat demand of Norway (that will be supplied by CH4)
+    - the normalized CH4 hourly resolved demand profile
+
+    Parameters
+    ----------
+    None.
+
+    Returns
+    -------
+    Norway_global_demand: Float
+        Yearly heat demand of Norway in MWh
+    neighbor_loads_t: pandas.DataFrame
+        Normalized CH4 hourly resolved demand profiles per neighbor country
+
+    """
+
+    cwd = Path(".")
+    target_file = (
+        cwd
+        / "data_bundle_egon_data"
+        / "pypsa_eur_sec"
+        / "2021-egondata-integration"
+        / "postnetworks"
+        / "elec_s_37_lv2.0__Co2L0-1H-T-H-B-I-dist1_2050.nc"
+    )
+
+    network = pypsa.Network(str(target_file))
+
+    # Set country tag for all buses
+    network.buses.country = network.buses.index.str[:2]
+    neighbors = network.buses[network.buses.country != "DE"]
+    neighbors = neighbors[
+        (neighbors["country"].isin(countries))
+        & (neighbors["carrier"] == "residential rural heat")
+    ].drop_duplicates(subset="country")
+
+    neighbor_loads = network.loads[network.loads.bus.isin(neighbors.index)]
+    neighbor_loads_t_index = neighbor_loads.index[
+        neighbor_loads.index.isin(network.loads_t.p_set.columns)
+    ]
+    neighbor_loads_t = network.loads_t["p_set"][neighbor_loads_t_index]
+    Norway_global_demand = neighbor_loads_t[
+        "NO3 0 residential rural heat"
+    ].sum()
+
+    for i in neighbor_loads_t.columns:
+        neighbor_loads_t[i] = neighbor_loads_t[i] / neighbor_loads_t[i].sum()
+
+    return Norway_global_demand, neighbor_loads_t

```

Comment:
It is kind of weird, that this function returs a TS for all countries and the sum for Norway